tests/topotests/multicast_pim6_dual_stack/test_multicast_pim6_dual_stack.py

```diff
@@ -0,0 +1,1952 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: ISC
+#
+# Copyright (c) 2023 by VMware, Inc. ("VMware")
+# Used Copyright (c) 2018 by Network Device Education Foundation,
+# Inc. ("NetDEF") in this file.
+#
+
+"""
+Following tests are covered to test multicast pim6 dual stack:
+
+1. Verify IPv4 and IPv6 mroute and upstream when mld and MLD join sent from same interface
```

Comment:
Did you mean IGMP and MLD here?

Reply:
yes Mobas, we are enabling both IGMP and MLD for these tests, thats why names as dual stack[v4/v6]. Thanks.

Reply:
What I mean to say here is, you need to change "when mld and MLD join" to "when IGMP and MLD join" in all the places.

Reply:
What I mean to say here is you need to change "when **mld** and MLD join" to "when **IGMP** and MLD join"